src/View.js

```diff
@@ -78,20 +78,19 @@ export default class View extends React.Component {
     const { rowClass, rowData, rowIndex, rowProps = {}, cells } = row;
 
     return (
-      <div
-        aria-rowindex={rowIndex + 2}
+      <button
         className={rowClass}
         data-label={[
           rowData.name,
           this.formatter.type(rowData),
           this.formatter.status(rowData),
         ].join('...')}
         key={`row-${rowIndex}`}
-        role="row"
+        type="button"
```

Comment:
What's the purpose of these changes? They significantly decrease a11y.

Reply:
The app results need to be rendering out something thats tab-able to fix https://issues.folio.org/browse/ERM-622. Hence changed this to a button. Are you specifically referring `role="row`? If so, thats coming through already via the `rowProps`

Reply:
Ahh OK. Does `aria-rowindex` get passed in as well?

Reply:
Yes, `aria-rowindex` gets passed as well